korman/nodes/node_responder.py

```diff
@@ -46,38 +45,38 @@ class PlasmaResponderNode(PlasmaVersionedNode, bpy.types.Node):
     default_state = IntProperty(name="Default State Index",
                                 options=set())
 
-    input_sockets = OrderedDict([
-        ("condition", {
+    input_sockets: dict[str, dict[str, any]] = {
+        "condition": {
             "text": "Condition",
             "type": "PlasmaConditionSocket",
             "spawn_empty": True,
-        }),
-    ])
+        },
+    }
 
-    output_sockets = OrderedDict([
-        ("keyref", {
+    output_sockets: dict[str, dict[str, any]] = {
+        "keyref": {
             "text": "References",
             "type": "PlasmaPythonReferenceNodeSocket",
             "valid_link_nodes": {"PlasmaPythonFileNode"},
-        }),
-        ("state_refs", {
+        },
+        "state_refs": {
             "text": "State",
             "type": "PlasmaRespStateRefSocket",
             "valid_link_nodes": "PlasmaResponderStateNode",
             "valid_link_sockets": "PlasmaRespStateRefSocket",
             "link_limit": 1,
             "spawn_empty": True,
-        }),
+        },
 
-        # This version of the states socket has been deprecated.
-        # We need to be able to track 1 socket -> 1 state to manage
-        # responder state IDs
-        ("states", {
+         # This version of the states socket has been deprecated.
+         # We need to be able to track 1 socket -> 1 state to manage
+         # responder state IDs
```

Comment:
This extra space might be introducing indentation issues. Comments in korman are aligned to the 4-space tab stops.